openapi3.yaml

```diff
@@ -1286,11 +1286,26 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/stageResponse'
+    createStagePayloadRequest:
+      allOf:
+        - $ref: '#/components/schemas/createStagePayload'
+        - type: object
+          properties:
+            isWaiting:
```

Comment:
consider a more descriptive name to suggest is means the starting status is  going to be waiting.
At least for me, isWaiting makes me think like a property to check if its waiting, not to create it as such.

Reply:
i renamed to startAsWaiting so now the original description sound to be good